### DISCUSSION TITLE
Slow startup time with bundled app

### DISCUSSION THREAD

**Discussion by weisbrja** [tauri-apps/tauri#11157] [Q&A] [answered]
I generated a basic Tauri + Preact + Typescript + Bun app using ```cargo create-tauri-app --rc```. (The ```--rc``` doesnt matter, I had the same problem without it.)

Is it normal for this kind of basic app to take 1-2s before it shows the html when using the bundled AppImage? If I just run ```bun run tauri dev```, the app has a much lower startup time, so I guess there must be something wrong with bundling.

Also, after opening, it first flashes a black and white screen before displaying the actual content, even in `dev` mode.

**Comment by FabianLars** [comment, marked as answer]
AppImages are _heavily_ compressed so i would assume that could be a large part of it.

Does the .deb or the binary (in target/release/) open faster?

**Reply by weisbrja** [reply]
Yes, the binary does open way faster. It's probably due to the compression. Since AppImages are super large, I didn't assume it even uses compression. But that makes sense, thanks.
